awx/sso/fields.py

```diff
@@ -1,3 +1,8 @@
+import collections
```

Comment:
Look!  Imports that this file **needs** to compile, but which it was _only getting_ by a coincidence of import order.  🤦‍♂️